integration-tests/package.json

```diff
@@ -53,5 +52,8 @@
     "rimraf": "^3.0.2",
     "shelljs": "^0.8.4",
     "typescript": "^4.2.3"
+  },
+  "dependencies": {
```

Comment:
Any reason why everything is in `devDependencies` instead of just `dependencies`?

Reply:
No real reason but since we're not distributing the package I prefer to keep everything in devDependencies.

Reply:
Actually the real reason is that we should maintain the habit of only using `dependencies` for things that need to be distributed with the package and since we're not distributing the integration tests we shouldn't have anything in `dependencies`.